### PR TITLE
[PB-535]: feat/return private folder shared with a user

### PR DESCRIPTION
**Pull request by cg27shokworks**

### CODE REVIEW

File: src/modules/private-share-folder/private-sharing.controller.ts [hunk 1]
```diff
@@ -17,7 +17,54 @@ import { Pagination } from 'src/lib/pagination';
...
+  })
+  @ApiQuery({
+    description: 'Order by',
+    name: 'orderBy',
+    required: false,
+    type: String,
+  })
+  @ApiOkResponse({ description: 'Get all folders shared with a user' })
+  @ApiBearerAuth()
+  async getReceivedFolders(
```
Comment (sg-gs) [inline src/modules/private-share-folder/private-sharing.controller.ts:45]: I found this name a bit confusing

File: src/modules/private-share-folder/private-sharing.repository.ts [hunk 2]
```diff
@@ -24,6 +24,31 @@ export class SequelizePrivateSharingRepository
     @InjectModel(FolderModel)
     private folderModel: typeof FolderModel,
   ) {}
+
+  async findSharedWithMePrivateFolders(
```
Comment (sg-gs) [inline src/modules/private-share-folder/private-sharing.repository.ts:28]: Too much business-related

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: src/modules/private-share-folder/private-sharing.repository.ts [hunk 3]
```diff
@@ -51,4 +57,32 @@ export class SequelizePrivateSharingRepository
...
+  ): Promise<Folder[]> {
+    const sharedFolders = await this.privateSharingFolderModel.findAll({
+      where: {
+        sharedWith: userUuid,
+      },
+      include: [
+        {
+          model: this.folderModel,
+          required: true,
+          foreignKey: 'folderUuid',
```
Comment (sg-gs) [inline src/modules/private-share-folder/private-sharing.repository.ts:75]: Why `folderUuid`?

Comment (cg27shokworks) [reply src/modules/private-share-folder/private-sharing.repository.ts]: Fixed, possibly a copilot completion error.

**Review by sg-gs — APPROVED** [review]

**Pull request merged**